api/document/js/example-doc.ts

```diff
@@ -0,0 +1,18 @@
+import {Node} from "prosemirror-model";
+
+import schema from "./schema";
+
+const EXAMPLE_DOC = {
+  type: 'doc',
+  content: [{
+    type: 'paragraph',
+    content: [{
+      type: 'text',
+      text: 'Hello, I am a paragraph.',
+    }],
+  }],
+};
+
+const exampleDoc = Node.fromJSON(schema, EXAMPLE_DOC);
+
+export default exampleDoc;
```

Comment:
Not that this file will be around for very long, but I think it's OK to export the actual document because [nodes in ProseMirror are immutable](https://prosemirror.net/docs/ref/#model.Node), yay!